content/docs/en/elements/components/list-view.md

```diff
@@ -4,53 +4,77 @@ apiRef: https://docs.nativescript.org/api-reference/classes/_ui_list_view_.listv
+`<ListView>` is a UI component that shows items in a vertically scrolling list. To set how the list shows its items, you can use the `<v-template>` component.
+
+```
+<ListView for="item in listOfItems" @itemTap="onItemTap"> // Iterates on every list item and attaches an itemTap event to it.
+  <v-template>
+    <Label :text="item.text" /> // Shows the list item label in the default color and stye.
+  </v-template>
+</ListView>
+```
 
 ---
 
+[> screenshots for=ListView <]
+
+## Using `<ListView>` with `<v-template>`
+
+The [`v-template` component](/en/docs/utilities/v-template) lets you determine how each list item is shown on the screen. 
+
+If you need to visualize one or more list items differently than the rest, you can enclose them in additional `<v-template>` blocks and use conditions. You can have as many `<v-template>` blocks as needed within one `<ListView>`.
+
 ```html
-<ListView for="item in listOfItems" @itemTap="onItemTap">
+<ListView for="item in listOfItems" @itemTap="onItemTap"> 
   <v-template>
-    <Label :text="item.text" />
+    <Label :text="item.text" /> 
   </v-template>
 
   <v-template if="$odd">
-    <Label :text="item.text" color="red" />
+    <Label :text="item.text" color="red" /> // For items with an odd index, shows the label in red. 
   </v-template>
 </ListView>
 ```
 
-In the above example the second template will be used if the item's index is odd.
-For convenience you can use `$index`, `$even` and `$odd` helpers, but as needed you can use any valid javascript expression for a `v-template`'s `if` condition.
+When you create conditions for `<v-template>`, you can use any valid JavaScript expression or any of the following helpers:
+
+* `$index`
+* `$even` 
+* `$odd`
 
-To learn more about the `v-template` component, head over to the [`v-template` documentation](/en/docs/utilities/v-template).
+## Using `<ListView>` with `v-for`
 
-The ListView does not loop through the items as you would expect when using a [`v-for`](https://vuejs.org/v2/guide/list.html#Mapping-an-Array-to-Elements-with-v-for) loop. Instead it only creates the necessary views to display the currently visible items on the screen, and when scrolling it reuses the views that are already off-screen. This concept is called view recycling, and is commonly used in mobile applications to improve performance. This is important because you can't rely on event listeners attached inside the `v-template`, instead you should use the `itemTap` event which contains the index of the tapped item, as well as the actual item from the list.
+When you use `<ListView>` with [`v-for`](https://vuejs.org/v2/guide/list.html#Mapping-an-Array-to-Elements-with-v-for), you need to use the default `itemTap` event instead of attaching other event listeners. The `itemTap` event contains the index of the tapped item and the actual item from the list.
```

Comment:
Can you change this back to the previous version, because the two are a bit different. What the old version means is that the ListView works differently that if you were using `v-for` (nativescript-vue will issue a warning when a `v-for` is used in a list view), and due to the different mechanism (view recycling) you can't rely on events inside a `v-template`. 

Although this may no longer be true, because if you don't have an `itemTap` event handler, then you can do custom events inside `v-template`, which is useful in cases where you may have a few actions for every list item (think of the like/retweet etc buttons in a tweet list view for example).